script/smt-data-export

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+
+NAME="$(basename $0)"
+OUTPUT_FILE=""
+TEMP_DIR=""
+TEMP_PATH=""
+EXPORT_SSL=0
+
+SMT_CONFIG=${SMT_CONFIG:-/etc/smt.conf}
+APACHE_CONFIG=${APACHE_CONFIG:-/etc/apache2}
+SSL_PATH=${SSL_PATH:-/var/lib/CAM/YaST_Default_CA}
+
+SMT_MYSQL_USER=""
+SMT_MYSQL_PASSWORD=""
+SMT_MYSQL_DATABASE=""
+
+# -- UTILITY ------------------------------------------------------------------
+
+function warn() { echo -ne "\033[31m$@\033[0m\n"; }
+
+# reads a configuration parameter from smt.conf
+function smt_conf_value() {
+  local key=$1
+  grep -oP "$key\s*=\s*\K.*" $SMT_CONFIG
```

Comment:
Quote them here with something like `printf "%q" "foo&bar"`, perhaps?

Reply:
We've tested it and bash takes care of escaping by itself, so no changes required :+1: